services/system/StagedTx/service/src/lib.rs

```diff
@@ -46,7 +46,7 @@ pub mod service {
     fn check_init() {
         let table: InitTable = InitTable::new();
         check(
-            table.get_index_pk().get(&()).is_some(),
+            table.get_index_pk().get(&&()).is_some(),
```

Comment:
Is this (and similar changes) needed?

Reply:
nope. residual from earlier implementations. good catch.
reverted